server/cmwell-dc/src/main/scala/cmwell/dc/stream/DataCenterSyncManager.scala

```diff
@@ -847,20 +851,17 @@ class DataCenterSyncManager(dstServersVec: Vector[(String, Option[Int])],
   }
 
   def runSyncingEngine(dcInfo: DcInfo): SyncerMaterialization = {
+    val transformationsStr = if (dcInfo.transformations.nonEmpty) s" with transformations: ${dcInfo.transformations.mkString("[", ",", "]")}" else ""
     logger.info(
-      s"Starting sync engine for data center id ${dcInfo.id} from location ${dcInfo.location}${dcInfo.positionKey
-        .fold("")(key => s" using position key $key")}${dcInfo.tsvFile
-        .fold("")(f => " and file " + f)}"
+      s"Starting sync engine for data center id ${dcInfo.id} from location " +
+        s"${dcInfo.location}${dcInfo.positionKey.fold("")(key => s" using position key $key")}${dcInfo.tsvFile.fold("")(f => " and file " + f)}" +
+        s"$transformationsStr"
```

Comment:
Don't you want to write something like: 
s"**transformations are:** $transformationsStr"

Reply:
It's already done. See line 854.